### PR TITLE
Install dependencies

### PR DESCRIPTION
**Pull request by petebacondarwin**
Simplifies #11322

### CODE REVIEW

**Comment by petebacondarwin** [comment]
How about this @Narretz ?

**Comment by Narretz** [comment]
Unfortunately `shell.rm` is causing the same "Path too long" errors on Windows as regulat `rm`. :/

**Comment by petebacondarwin** [comment]
OK, so `rimraf.sync` it is then, eh?

**Comment by Narretz** [comment]
Looks like it! I'll test it later today, let's hope it's fast (although that shouldn't really matter because npm install is so slow)

**Comment by Narretz** [comment]
I tested it twice and it looks good!

File: Gruntfile-test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+var npmInstall = require('./scripts/npm/install-dependencies');
+
+module.exports = function(grunt) {
+
+  grunt.registerTask('npm-install', function() {
+    npmInstall.installDependencies();
+  });
+};
```
Comment (petebacondarwin) [inline Gruntfile-test.js]: We need to remove this once we are finished testing

**Pull request closed without merging**